### ISSUE TITLE
Build: check-commit length check is off by 1 for single line commits

### ISSUE THREAD

**Issue by Mardak** [eslint/eslint#6109]
**What did you do? Please include the actual source code causing the issue.**
```sh
git commit -m 'Fix:56789012345678901234567890123456789012345678901234567890(fixes #9012)'
npm run check-commit
```

**What did you expect to happen?**
`First line of commit message must not exceed 72 characters`

**What actually happened? Please include the actual, raw output from ESLint.**
no error

The commit message above is 73 characters long, so it should have failed the test.

For #3706 @platinumazure added the check in https://github.com/eslint/eslint/commit/c8370ade8305dd3bc0bb86104e709b35e657c796 using `slice`. However, when there's no newline, `indexOf` returns `-1`, so `slice` returns a commit message without the ending ")" (of "(fixes #9012)".

**Comment by platinumazure** [comment]
Oh heck. For some reason I thought `slice` was inclusive. Thanks for fixing
this!
